src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/IntegerCopyNumberState.java

```diff
@@ -0,0 +1,90 @@
+package org.broadinstitute.hellbender.tools.copynumber.gcnv;
+
+import htsjdk.variant.variantcontext.Allele;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+
+/**
+ * This class represents integer copy number states. It is intended to be used as states of a
+ * hidden Markov model.
+ */
+public final class IntegerCopyNumberState  {
+
+    /**
+     * Integer value of the represented copy number state
+     */
+    private final int copyNumber;
+
+    /**
+     * A string representation of this copy number state (used for creating human-readable copy number call lists)
+     */
+    private final String callString;
+
+    /**
+     * An allele representation of this copy number state (used for VCF creation)
+     */
+    private final Allele allele;
```

Comment:
remove

Reply:
Done

Reply:
Will do for the next release as we discussed

Reply:
Done